src/clarity-angular/layout/tabs/tabs.spec.ts

```diff
@@ -67,88 +68,24 @@ describe("Tabs", () => {
-
-        expect(instance.tabLinks[0].id).toMatch(/clr-tabs-[0-9]+-tab-0/);
-        expect(instance.tabContents[0].id).toMatch(/clr-tabs-[0-9]+-content-0/);
-        expect(linkElements[0].textContent).toMatch(/Tab1/);
-        expect(contentElements[0].textContent).toMatch(/Content1/);
-
-        expect(instance.tabLinks[1].id).toMatch(/clr-tabs-[0-9]+-tab-1/);
-        expect(instance.tabContents[1].id).toMatch(/clr-tabs-[0-9]+-content-1/);
-        expect(linkElements[1].textContent).toMatch(/Tab2/);
-        expect(contentElements[1].textContent).toMatch(/Content2/);
-
-        expect(instance.tabLinks[2].id).toMatch(/clr-tabs-[0-9]+-tab-2/);
-        expect(instance.tabContents[2].id).toMatch(/clr-tabs-[0-9]+-content-2/);
-        expect(linkElements[2].textContent).toMatch(/Tab3/);
-        expect(contentElements[2].textContent).toMatch(/Content3/);
-
-        expect(instance.tabLinks[3].id).toMatch(/clr-tabs-[0-9]+-tab-3/);
-        expect(linkElements[3].textContent).toMatch(/Tab4/);
-    });
+        expect(compiled.querySelectorAll("section").length).toEqual(0);
 
-    it("activates the matching tab content when a tab is selected", () => {
-        instance.selectTab(instance.tabLinks[2]);
-        expect(instance.tabLinks[0].active).toBe(false);
-        expect(instance.tabLinks[1].active).toBe(false);
-        expect(instance.tabLinks[2].active).toBe(true);
-        expect(instance.tabContents[0].active).toBe(false);
-        expect(instance.tabContents[1].active).toBe(false);
-        expect(instance.tabContents[2].active).toBe(true);
+        instance.ngAfterContentInit();
+        fixture.detectChanges();
+        expect(compiled.querySelectorAll("button.nav-link").length).toEqual(4);
+        expect(compiled.querySelectorAll("section").length).toEqual(1);
     });
 
-    it("does't activate any content when an orphan tab is selected", () => {
-        instance.selectTab(instance.tabLinks[3]);
-        expect(instance.tabLinks[0].active).toBe(false);
-        expect(instance.tabLinks[1].active).toBe(false);
-        expect(instance.tabLinks[2].active).toBe(false);
-        expect(instance.tabLinks[3].active).toBe(true);
-        expect(instance.tabContents[0].active).toBe(false);
-        expect(instance.tabContents[1].active).toBe(false);
-        expect(instance.tabContents[2].active).toBe(false);
+    it("sets the first tab as active by default", () => {
+        let tabs = instance.tabChildren.toArray();
+        expect(instance.activeTab).toEqual(tabs[0]);
     });
 
-    it("emits a clrTabSelected event when a tab is clicked", (done: any) => {
-        instance.currentTabLinkChanged.subscribe((selectedTab: TabLink) => {
-            expect(selectedTab).toEqual(instance.tabLinks[0]);
-            done();
-        });
-
-        instance.selectTab(instance.tabLinks[0]);
+    it("sets the designated tab as active", () => {
+        let tabs = instance.tabChildren.toArray();
+        expect(instance.activeTab).toEqual(tabs[0]);
```

Comment:
I think this functionality is necessary. Having the first tab displayed by default is Tabs 101, that's something our users don't want to have to worry about. We need to put it back and solve the EHCAIWC error.